tools/c7n_tencentcloud/c7n_tencentcloud/resources/ami.py

```diff
@@ -0,0 +1,106 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+import itertools
+
+from c7n.utils import type_schema
+from c7n.filters import Filter, AgeFilter
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod
+
+
+@resources.register("ami")
+class AMI(QueryResourceManager):
+    """ami"""
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "ImageId"
+        endpoint = "cvm.tencentcloudapi.com"
+        service = "cvm"
+        version = "2017-03-12"
+        enum_spec = ("DescribeImages", "Response.ImageSet[]", {})
+        metrics_instance_id_name = "VpcId"
+        paging_def = {"method": PageMethod.Offset, "limit": {"key": "Limit", "value": 20}}
+        resource_prefix = "instance"
+        taggable = True
+
+    def get_resource_query_params(self):
+        """
+        https://cloud.tencent.com/document/api/213/15715
+        only query image-type = PRIVATE_IMAGE
+        """
+        config_query = self.data.get("query", [])
+        params = {
+            "Filters": [
+                {
+                    "Name": "image-type",
+                    "Values": ["PRIVATE_IMAGE"]
+                }
+            ]
+        }
+        for it in config_query:
+            params.update(it)
+
+        return params
+
+
+@AMI.filter_registry.register('image-age')
+class ImageAgeFilter(AgeFilter):
```

Comment:
we don't really need this, or do this anymore unless its handling special cases, a simple value attribute with value_type are suffices, worth putting a doc example if you think its needed, but in either case we can omit the dedicated filter.

Reply:
done